### PR TITLE
[v6r14] Fixing the tests

### PR DESCRIPTION
**Pull request by fstagni**
As per https://travis-ci.org/DIRACGrid/DIRAC/builds/122505590

### CODE REVIEW

File: Core/Utilities/test/Test_File.py [hunk 1]
```diff
@@ -131,10 +132,13 @@ def testGetCommonPath( self ):
 
     # this folder and its parent
     filesList = [ os.path.abspath("..") + os.sep + x for x in os.listdir("..") ] + self.filesList
+    #FIXME: the following lines are here temporary for debugging
+    print 'self.fileList', self.filesList
+    print 'fileList', filesList
     self.assertEqual( getCommonPath( filesList ), os.path.abspath("..") )
```
Comment (petricm) [inline Core/Utilities/test/Test_File.py]: os.path.abspath(".")?

Comment (fstagni) [inline Core/Utilities/test/Test_File.py]: If I put that it fails when I run locally. The test itself is correct, I think there's a protection in Jenkins on listing a directory one level up. I am either removing or changing it.

Comment (fstagni) [inline Core/Utilities/test/Test_File.py]: Done, it should run now.

**Comment by fstagni** [comment]
Any "review OK" here?

**Pull request merged**